cryptography/src/bls12381/primitives/poly.rs

```diff
@@ -268,8 +268,8 @@ impl<C: Element> Poly<C> {
 }
 
 /// Returns the public key of the polynomial (constant term).
-pub fn public(public: &Public) -> group::Public {
-    *public.constant()
+pub fn public(public: &Public) -> &group::Public {
```

Comment:
In other places in the codebase, we now return a reference where it makes sense (leaving it up to the caller whether or not they should clone.